vispy/app/backends/_qt.py

```diff
@@ -114,7 +114,7 @@ def _get_event_xy(ev):
     _check_imports('PySide6')
     if not USE_EGL:
         from PySide6.QtCore import __version__ as QT_VERSION_STR
-        if LooseVersion(QT_VERSION_STR) >= '6.0.0':
```

Comment:
packaging does not support direct comparison to strings. So the hardest part is remembering to do this everywhere.